app/ext/api_avatar.py

```diff
@@ -1,13 +1,14 @@
 import requests
 import random
 
-colors = ['amber', 'blue', 'blueGrey', 'brown', 'cyan', 'deepOrange', 'deepPurple', 'green', 'grey', 'indigo', 'lime', 'orange', 'pink', 'purple', 'red', 'teal']
+colors = ['amber', 'blue', 'blueGrey', 'brown', 'cyan', 'deepOrange', 'deepPurple', 'green', 'grey', 'indigo', 'lime',
+          'orange', 'pink', 'purple', 'red', 'teal']
 
 
 def create_avatar(name, lastname):
     color_random = random.choice(colors)
-    response = requests.get(f"https://avatars.dicebear.com/api/initials/{name}%20{lastname}.svg?options[radius]=50&options[backgroundColors][]={color_random}")
+    response = requests.get(
+        f"https://avatars.dicebear.com/api/initials/{name}%20{lastname}."
+        f"svg?options[radius]=50&options[backgroundColors][]={color_random}")
```

Comment:
Nice to have to see everything in my screen and not have to scroll left and right. IDES can help you in that you just press ENTER in the middle of the string and IDE does the rest